subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -495,6 +497,16 @@ static int wifi_scan_args_to_params(const struct shell *sh,
 				return -ENOEXEC;
 			}
 			break;
+		case 'a':
+			val = atoi(optarg);
+
+			if ((val <= 5) || (val >= 1000)) {
```

Comment:
These should be inclusive, also, as per 802.11-20202 these are for OBSS scan, normal scan doesn't seem to have any limits.

Reply:
Yes, even though the specification specifies it for OBSS scan they seem to be sensible even for non-OBSS scans, hence using them here.